src/api.rs

```diff
@@ -339,6 +339,9 @@ impl Config {
       aom_dsp_rtcd();
     }
 
+    // initialize with temporal delimiter
+    let packet_data = vec![0x12, 0x00];
```

Comment:
You could hide this inside a constant temporal OBU header somewhere, so you define it once and use it twice.